Dockerfile

```diff
@@ -33,6 +33,12 @@ ARG NEXT_PUBLIC_DOCSEARCH_APP_ID
 ARG NEXT_PUBLIC_DOCSEARCH_API_KEY
 ARG NEXT_PUBLIC_DOCSEARCH_INDEX_NAME
 
+# Building different architectures via emulation is slow with Yarn
+# This increases the timeout period so it can properly download dependencies
+# Value is in milliseconds and is set to 60 minutes
+# To increase/decrease you can override via --build-arg YARN_TIMEOUT=x in your build command
+ARG YARN_TIMEOUT=3600000
```

Comment:
Does this mean a build might take close to one hour when this action runs?

Reply:
The longest I've observed so far is around 30 minutes, I just increased it so we have a buffer since it is known to take a while. If GitHub supports arm64 runners in the future (the ones we use and not the large runners in which they now support arm), we'd be able to build the arm64 image in minutes.